### DISCUSSION TITLE
buttonTo() does not accept "id" or "class" attributes

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1352] [Q&A] [answered]
I've tried adding both a class and an id attribute to a button using the buttonTo() function.  It is not working for me.

**Comment by MvdO79** [comment, marked as answer]
Did you use: "inputId="edit-button", inputClass="edit-button-class"?

**Reply by cfcoder** [reply]
No, where is that documented?

**Reply by neokoenig** [reply]
There's an example on https://api.cfwheels.org/v2.5.0#controller.buttonto - but yeah, it's not implicitedly inplied in the arguments list

**Reply by cfcoder** [reply]
May I suggest this be documented with higher visibility, like for all functions it applies to, add a Note or include it in the attribute list of options.  I know other functions accept "id" and "class". So it's not clear when to use one vs. the other. I have seen a note for a function that accepts id and class but don't know if all functions that support them have that same note or not.

**Reply by MvdO79** [reply]
Good idea, will pick it up 👍

**Comment by zainforbjs** [comment]
Answered and closing the issues, Thank you @MvdO79 for your support.
Thanks again.

**Reply by MvdO79** [reply]
Your welcome!
